backoff/strategy_test.go

```diff
@@ -0,0 +1,62 @@
+package backoff
+
+import (
+	"time"
+
+	"github.com/dogmatiq/linger"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("func Exponential()", func() {
+	It("returns a strategy that backs-off exponentially", func() {
+		strategy := Exponential(3 * time.Second)
+
+		Expect(strategy(nil, 4)).To(Equal(48 * time.Second))
+		Expect(strategy(nil, 5)).To(Equal(96 * time.Second))
+	})
+
+	It("panics if the unit is zero", func() {
+		Expect(func() {
+			Exponential(0)
+		}).To(Panic())
+	})
+
+	It("panics if the unit is negative", func() {
+		Expect(func() {
+			Exponential(-1)
+		}).To(Panic())
+	})
+})
+
+var _ = Describe("func Constant()", func() {
+	It("returns a strategy that returns a fixed duration", func() {
+		strategy := Constant(3 * time.Second)
+
+		Expect(strategy(nil, 4)).To(Equal(3 * time.Second))
+		Expect(strategy(nil, 5)).To(Equal(3 * time.Second))
+	})
+})
+
+var _ = Describe("func Linear()", func() {
+	It("returns a strategy that returns a fixed duration", func() {
```

Comment:
I don't think this test description should be "fixed" also?
```suggestion
	It("returns a strategy that returns a linearly increasing duration", func() {
```